src/CMakeLists.txt

```diff
@@ -296,5 +296,12 @@ if(ENABLE_SWIFT)
             DESTINATION
               ${INSTALL_TARGET_DIR})
   endif()
+
+  if(NOT BUILD_SHARED_LIBS)
+    install(FILES
+              ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_${library_kind}_LIBRARY_PREFIX}DispatchStubs${CMAKE_${library_kind}_LIBRARY_SUFFIX}
```

Comment:
Since we are in a `NOT BUILD_SHARED_LIBS` case, we can simplify this:

`${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}/DispatchStubs${CMAKE_STATIC_LIBRARY_SUFFIX}`

which can be simplified to:

`$<TARGET_FILE:DispatchStubs>`